jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/src/test/java/org/jbpm/test/container/webspherefix/WebSphereFixedJtaPlatform.java

```diff
@@ -0,0 +1,41 @@
+package org.jbpm.test.container.webspherefix;
```

Comment:
Could you please add license header for this file and this will be ready to merge